src/components/Article.jsx

```diff
@@ -26,7 +25,7 @@ console.log(user)
         fetchComments(article_id).then(({ comments }) => {
             setComments(comments)
         })
-    }, [])
+    }, [comments])
```

Comment:
In this useEffect() you are setting comments and your dependency is also comments, which has resulted in an infinite loop. Please sort this out.